core/templates/dev/head/pages/about-page/about-page.directive.html

```diff
@@ -286,8 +290,8 @@ <h2 translate="I18N_ABOUT_PAGE_CREDITS_TAB_HEADING"></h2>
                 <li>Kevin Thomas</li>
                 <li>Kiran Konduru</li>
                 <li>Koji Ashida</li>
-                <li>Kristin Anthony</li>
```

Comment:
Hm! It might be worth checking with @ankita240796 why this wasn't caught before. (Thanks for fixing!)

Reply:
This was an automatic change :p, I'm worried why it didn't happen in the earlier release (The auto changelog PR generator script is working from last 3+ months). 

Maybe the script sorts only the section it updates (Like in our case the "K" section).

Reply:
Yeah, that's a possible explanation. Might be worth checking with @ankita240796 !